projects/storefrontlib/src/lib/my-account/update-password/components/update-password-form/update-password-form.component.html

```diff
@@ -0,0 +1,69 @@
+<form
+  (submit)="onSubmit()"
+  [formGroup]="form"
+  class="cx-update-password-component "
+>
+  <div class="form-group">
+    <label>
+      <span class="label-content">Old Password</span>
+      <input
+        class="form-control"
+        [class.is-invalid]="isNotValid('oldPassword')"
+        type="password"
+        name="oldPassword"
+        placeholder="Old Password"
+        formControlName="oldPassword"
+      />
+      <div class="invalid-feedback" *ngIf="isNotValid('oldPassword')">
+        <span>Old password is required.</span>
+      </div>
+    </label>
+    <label>
+      <span class="label-content">New Password</span>
+      <input
+        class="form-control"
+        [class.is-invalid]="isNotValid('newPassword')"
+        type="password"
+        name="newPassword"
+        placeholder="New Password"
+        formControlName="newPassword"
+      />
+      <div class="invalid-feedback" *ngIf="isNotValid('newPassword')">
+        <span
+          >Password must be six characters minimum, with one uppercase letter,
+          one number, one symbol</span
+        >
+      </div>
+    </label>
+  </div>
+  <div class="form-group">
+    <label>
+      <span class="label-content">Confirm New Password</span>
+      <input
+        class="form-control"
+        [class.is-invalid]="isPasswordConfirmNotValid()"
+        type="password"
+        name="newPasswordConfirm"
+        placeholder="Confirm Password"
+        formControlName="newPasswordConfirm"
+      />
+      <div class="invalid-feedback" *ngIf="isPasswordConfirmNotValid()">
+        <span>Both password must match</span>
+      </div>
+    </label>
+  </div>
+  <div class="form-group row">
+    <div class="col-lg-6 col-md-12">
+      <button
+        class="btn btn-block btn-secondary"
+        type="button"
+        (click)="onCancel()"
+      >
+        Cancel
+      </button>
+    </div>
+    <div class="col-lg-6 col-md-12">
+      <button class="btn btn-block btn-primary" type="submit">Save</button>
```

Comment:
On a mobile view, the buttons don't have space between them:
<img width="552" alt="Screen Shot 2019-04-07 at 18 01 51" src="https://user-images.githubusercontent.com/11561723/55690428-99496500-595f-11e9-9d1a-bbdb0b1c3273.png">

Reply:
fixed.